executor/scripts/build_local.py

```
@@ -125,19 +125,130 @@ def restore_version_source(original_content: str) -> None:
+                zf.extract(binary_zip_path, temp_dir)
+                binary_path = temp_dir / binary_zip_path
+
+                print(
+                    f"Extracted Claude CLI binary: {binary_path} ({binary_path.stat().st_size / 1024 / 1024:.1f} MB)"
+                )
+                return (str(binary_path), "claude_agent_sdk/_bundled")
+            else:
+                print(f"Error: {binary_zip_path} not found in wheel")
+                return None
+
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)